x-pack/plugins/security_solution/public/detections/components/alerts_table/actions.tsx

```diff
@@ -344,9 +345,7 @@ export const buildEqlDataProviderOrFilter = (
         'signal.group.id',
         ecs.reduce<string[]>((acc, ecsData) => {
           const alertGroupIdField = getField(ecsData, ALERT_GROUP_ID);
-          const alertGroupId = alertGroupIdField?.length
-            ? alertGroupIdField[0]
-            : 'unknown-group-id';
+          const alertGroupId = alertGroupIdField?.length ? alertGroupIdField : 'unknown-group-id';
```

Comment:
`alertGroupIdField ?? 'unknown-group-id'`?

Reply:
+1 to @madirey suggestion.